skimage/util/tests/test_dtype.py

```diff
@@ -109,7 +109,7 @@ def test_copy():
 
 def test_bool():
     img_ = np.zeros((10, 10), bool)
-    img8 = np.zeros((10, 10), np.bool8)
+    img8 = np.zeros((10, 10), np.bool_)
```

Comment:
Should `np.bool_` just be `bool`?

Reply:
I think they are all equivalent. They do have different entries here, though: https://github.com/scikit-image/scikit-image/blob/f11c27f13bd34720b91f7ebd4b4be810fe5493c3/skimage/util/dtype.py#L25-L27

so I think that is why there was both bool and np.bool8 separately here previously. I just changed `bool8` to `bool_` since that one is not deprecated.